import{_ as c,o as n,c as s,a as e,d as a}from"./index-0e88ad33.js";const t={},o={class:"container"},r=e("h1",null,"Manejo de excepciones",-1),i=e("hr",null,null,-1),l=e("p",null,[a("Probar y verificar que se lancen excepciones esperadas es una parte crucial de las pruebas unitarias. En JUnit5, puedes lograr esto de manera efectiva utilizando la anotación "),e("code",null,"@Test"),a(" junto con el método "),e("code",null,"assertThrows"),a(". Esta combinación te permite especificar una excepción esperada y luego verificar si se lanza correctamente durante la ejecución de tu prueba.")],-1),d=[r,i,l];function u(_,p){return n(),s("div",o,d)}const h=c(t,[["render",u]]);export{h as default};
